lib/tasks/print_statuses.rake

```diff
@@ -0,0 +1,30 @@
+desc "Print statuses of Tasks, Issues & Risks"
+task :print_statuses => :environment do
+  puts "*"*89, "TASKS", "*"*89
+  Task.all.each do |task|
+    is_overdue = false
+    is_overdue = task.progress < 100 && (task.due_date < Date.today) if !task.ongoing && !task.on_hold && !task.draft
+    planned = false
+    planned = true if !task.draft && task.start_date > Date.today
+    in_progress = false
+    in_progress = true if !task.draft && !task.on_hold && !is_overdue && !task.ongoing && task.progress >= 100 && task.start_date < Date.today
+
+    puts "Task ID: #{task.id}, Draft: #{task.draft}, OnHold: #{task.on_hold}, OnGoing: #{task.ongoing}, Overdue: #{is_overdue}, Planned: #{planned}, InProgress: #{in_progress}"
```

Comment:
If possible print this in CSV format so that it can be easily modify and download from server. The output will be long and will not be able to copy I assume.

Reply:
Good suggestion but the requirement from Juan was just to print the results.